roboticslogger/logger.py

```diff
@@ -44,8 +46,10 @@ def run(self, conn):
             elif msg[0] == "crit":
                 self.crit(msg[1])
             else:
-                error("Cannot log properly msg.")
+                self.err("Failed to log message.")
```

Comment:
LOL this is the case where the pipe receives some garbage input (ie a list with an improperly named first element.)